ludwig/visualize.py

```diff
@@ -1241,7 +1241,7 @@ def learning_curves(
 
                 validation_stats = []
                 for learning_stats in train_stats_per_model_list:
-                    if VALIDATION in learning_stats:
+                    if VALIDATION in learning_stats and len(learning_stats[VALIDATION]) > 0:
```

Comment:
I think checking for `output_feature_name in learning_stats[VALIDATION]`
```suggestion
                    if VALIDATION in learning_stats and output_feature_name in learning_stats[VALIDATION]:
```
is a bit more self-documenting.

Reply:
@dantreiman   Thank you for the comment.  I incorporated your suggestion.